### PR TITLE
Ecoworld frontend job added

### PR DESCRIPTION
**Pull request by nikoskleidis**

### CODE REVIEW

File: jobs/README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Front End Developers @ [Ecoworld SA – Greekferries Club](https://greekferries-club.gr)
+Ecoworld SA is looking for 2 Front End Developers (Junior & Senior) for full-time, in-house employment
+to work on various innovative online-travel related projects
```
Comment (gsaslis) [inline jobs/README.md]: IIRC for a list to start, you need a blank line before the list items that start with `*`

File: jobs/README.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ In this section you can find some job openings listed by our community members.
 
 __Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
 
+## Front End Developers @ [Ecoworld SA – Greekferries Club](https://greekferries-club.gr)
+Ecoworld SA is looking for 2 Front End Developers (Junior & Senior) for full-time, in-house employment
```
Comment (gsaslis) [inline jobs/README.md]: > in-house

might be worth mentioning where this is (e.g. Heraklion, Athens, etc.)

Comment (nikoskleidis) [reply jobs/README.md]: Good point. Added Heraklion on the title

File: jobs/README.md (hunk 1 above)
Comment (nikoskleidis) [reply jobs/README.md]: I didn't want to start a list. The new line was redundant though.

File: jobs/README.md (hunk 2 above)
Comment (nikoskleidis) [reply jobs/README.md]: I forced pushed in order not to have multiple commits. The change I made is not reflected here as I can see...

File: jobs/README.md (hunk 1 above)
Comment (gsaslis) [reply jobs/README.md]: please ignore. I was probably confusing asciidoc, with markdown.

**Pull request merged**